k8s/namespaces/nfdiv/nfdiv-cron-progress-held-cases/nfdiv-cron-progress-held-cases.yaml

```diff
@@ -1,16 +1,16 @@
 apiVersion: helm.fluxcd.io/v1
 kind: HelmRelease
 metadata:
-  name: nfdiv-cron-system-progress-held-cases-task
+  name: nfdiv-cron-progress-held-cases
 spec:
-  releaseName: nfdiv-cron-system-progress-held-cases-task
+  releaseName: nfdiv-cron-progress-held-cases
   chart:
     git: git@github.com:hmcts/nfdiv-cron
     ref: 0.0.7
     path: nfdiv-cron
   values:
     job:
-      image: hmctspublic.azurecr.io/nfdiv/case-api:prod-4da7bf9-20210831134705 #{"$imagepolicy": "flux-system:nfdiv-case-api"}
+      image: hmctspublic.azurecr.io/nfdiv/case-api:prod-00fe383-20210826060439  #{"$imagepolicy": "flux-system:nfdiv-case-api"}
```

Comment:
This PR is changing the image to an older image, can you check this file

Reply:
updated to latest image but we will also rely on image automation